api/db/activity/costAllocation.js

```diff
@@ -0,0 +1,21 @@
+module.exports = {
+  apdActivityCostAllocation: {
+    tableName: 'activity_cost_allocation',
+
+    activity() {
+      return this.belongsTo('apdActivity');
+    },
+
+    toJSON() {
+      return {
+        id: this.get('id'),
+        entity: this.get('entity'),
+        percent_of_cost: this.get('percent_of_cost')
```

Comment:
In some of the models, we convert to camel case before sending it to JSON and in others we don't.  I'm fine either way, but we should pick one and harmonize.